.github/workflows/unittests.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [12.x, 14.x, 16.x]
+        node-version: [16.x, 18.x, 20.x]
```

Comment:
I made two changes here:

- Dropped `12.x` and `14.x`, because they are [no longer supported](https://github.com/nodejs/Release/blob/6a881a11363eb601f17b2d46e3f1664db5ce2a3b/README.md#end-of-life-releases)
- Added `18.x` and `20.x` because those are latest versions that are, or will become, LTS. Not strictly related to this change, but thought it made sense to do now.